js/modal.js

```diff
@@ -0,0 +1,109 @@
+import modalData from './modalData.js';
+
+document.querySelector('.grid-box').innerHTML = modalData
+  .map(
+    (data, index) => `
+<div class="box">
+<img src="${data.snapshoot}" alt="Project image" class="banner">
+<div class="desktop-box">
+  <h3>${data.projectName}</h3>
+  <ul class="dot">
+    <li class="dot1">${data.frame[0]}</li>
+    <li><img src="${data.dot}"></li>
+    <li class="dot2">${data.frame[1]}</li>
+    <li><img src="${data.dot}"></li>
+    <li class="dot3">${data.frame[2]}</li>
+  </ul>
+  <p class="box-text">
+    ${data.para}
+  </p>
+  <ul class="tagul mediaTag">
+    <li class="tag">${data.technologies[0]}</li>
+    <li class="tag">${data.technologies[1]}</li>
+    <li class="tag">${data.technologies[2]}</li>
+  </ul>
+  <button
+    class="sm-btn"
+    aria-label="Enter here to see this project."
+    id="project_${index}"
+    data-target="modal">
+    ${data.smBtnText}
+  </button>
+</div>
+</div>
+`,
+  )
+  .join('');
+
+const displayModal = (data) => {
+  const modalSection = document.querySelector('.modal');
+  modalSection.innerHTML = `
+  <div class="overlay overlay-hid"></div>
+  <div class="modal-hid__content">
+    <div class="modal-hid__content__upper">
+      <div class="modal-hid__content__upper__head">
+     <h3>${data.projectName}</h3>
+      <button class="closeBtn">&times;</button>
+      </div>
+        <ul class="dot">
+          <li class="dot1">${data.frame[0]}</li>
+          <li><img src="${data.dot}"></li>
+          <li class="dot2">${data.frame[1]}</li>
+          <li><img src="${data.dot}"></li>
+          <li class="dot3">${data.frame[2]}</li>
+        </ul>
+        <div class="popPic-container">
+          <img src="${data.popPic}" alt="Project image" class="banner popPic">
+        </div>
+    </div>
+    <div class ="popPart">
+      <div class="modal-hid__content__mid">
+           <p class="box-text">
+           ${data.description}
+            </p>
+      </div>
+        <div class ="popPart2">
+         <ul class="tagul popTag">
+              <li class="tag">${data.technologies[0]}</li>
+              <li class="tag">${data.technologies[1]}</li>
+               <li class="tag">${data.technologies[2]}</li>
+         </ul>
+     
+         <img src="${data.separator}" class="sep">
+            <div class="modal-hid__content__lower">
+               <button class="sm-btn lowerBtn">See live <img src="${data.liveLogo}"></button>
+              <button class="sm-btn lowerBtn">See Source <img src="${data.sourceLogo}"></button>
+            </div>
+        </div>
+    </div>
+</div>
+  `;
```

Comment:
Nice job implementing the required functionalities of the popup window👍 but I think we can still make our popup window look more like the design for both desktop and mobile👍  

your popup (desktop)
![image](https://user-images.githubusercontent.com/64643995/216398702-598adeba-2601-4d9b-8993-c854faec63d0.png)


design(desktop)
![Screenshot from 2023-02-02 18-26-04](https://user-images.githubusercontent.com/64643995/216398489-f86ca1e0-1af8-4498-9e4a-c249769e68d6.png)

your project (mobile)
![image](https://user-images.githubusercontent.com/64643995/216399067-fb3052be-fe36-427b-8179-809da3c0004b.png)

design (mobile)
![image](https://user-images.githubusercontent.com/64643995/216399228-f2e402d9-743a-4f23-a6dd-b4ef2567c1ab.png)

kindly take note of the appearance, spacings, and image width and make the necessary changes👍